server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ExpressionAnalyzer.java

```diff
@@ -1324,8 +1324,8 @@ public static boolean isDeviceViewNeedSpecialProcess(Expression expression) {
     } else if (expression instanceof UnaryExpression) {
       return isDeviceViewNeedSpecialProcess(((UnaryExpression) expression).getExpression());
     } else if (expression instanceof FunctionExpression) {
-      if (((FunctionExpression) expression).isBuiltInFunction()
-          && BuiltinFunction.DEVICE_VIEW_SPECIAL_PROCESS_FUNCTIONS.contains(
+      if (((FunctionExpression) expression).isBuiltInScalarFunction()
```

Comment:
Previously, we named it as BuiltInScalarFunction, but I've forgot why Weihao Li changed it, you need to discuss with him @Wei-hao-Li

Reply:
we decided to use BuiltinScalar.